### DISCUSSION TITLE
Copilot Labs Error: Please check that the main GitHub Copilot extension is installed and running and try again.

### DISCUSSION THREAD

**Discussion by Kokkini** [community/community#21219] [Copilot Conversations]
I have the main Github Copilot extension and it's working normally. However, when I install and use Copilot Labs it gives me the error: "Something went wrong. Please check that the main GitHub Copilot extension is installed and running and try again."
Does anyone know how to resolve this?

**Comment by JaimeTR** [comment]
I installed GitHub Copilot Nightly, and it fixed the problem for me.
Before installing GitHub Copilot Nightly, please uninstall or disable Github copilot and install the extension from https://marketplace.visualstudio.com/items?itemName=GitHub.copilot-nightly

If this answer is helpful, please 'Mark as answer'. :)
